dev/VSIX/Extension/ExtensionPack.vsext

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Read about creating extension packs at https://aka.ms/vsextpack",
+  "version": "1.0.0.0",
+    "extensions": [
+        {
+            "vsixId": "Microsoft.SingleProjectMSIXPackagingTools",
```

Comment:
I don't see the connection between this vsixId and the two single-project templates - how does this work?